packages/react-app/src/components/BadgesPaginatedSection.tsx

```diff
@@ -92,43 +93,43 @@ export default function BadgesPaginatedSection({
               <AddressedCard badges={badges} etherscanRef={etherscanRef} />
             </Grid>
           ) : groupedRewards && groupedRewards.length > 0 ? (
-            groupedRewards.map(event => {
+            groupedRewards.map((eventB: any) => {
```

Comment:
can you type the `event` object and also the the return of `tokensData`? that will help see more clearly what's going on.